port/zephyr/src/clock.c

```diff
@@ -29,6 +29,12 @@ oc_clock_time(void)
   return k_uptime_get();
 }
 
+oc_clock_time_t
+oc_clock_time_monotonic(void)
+{
+  return k_uptime_get();
```

Comment:
I don't even know how to compile zephyr, so I didn't want to add a new function, I guess we can return -1 and if the zephyr implementation ever gets updated it will get changed.